archive/rvv-intrinsic-rfc.md

```diff
@@ -1,42 +1,62 @@
 # RISC-V "V" Vector Extension Intrinsics
 
-- [Introduction](#introduction)
```

Comment:
This is archived and users should not refer here for intrinsic specifications or behavior. Lets remove change for this file?